notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<p>It would be cool to quickly show how you would update the dataset based on the report (e.g. remove all of the "bad" examples, or add a column indicating which ones are good-to-use, and which ones are not).  I imagine one would want to run such cleanup on schedule, and somehow integrate the results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Well, Cleanlab is the package that helps identify these issues. One can simply delete the <code>near_duplicates</code> or <code>outliers</code> from dataframe and export the CSV. Cleanlab calls it <code>Cleanset</code> like Cleaned <code>Dataset</code>  I'd want the user to take a look and objectively delete the data points according to their choice.</p><p> </p><p>As per the goal of the project, it is aimed at showcasing the problems within dataset , it would be a bit difficult to integrate package like a workflow in my opinion (could be done using a GH Action or so!) </p><p> </p><p>Hence also the last paragraph, Cleanlab Studio helps with the necessary UI and longterm solution in maintaining the datasets throughout. </p>